x-pack/plugins/security_solution/public/management/components/endpoint_responder/endpoint_response_actions_console_commands.tsx

```diff
@@ -62,21 +92,58 @@ const COMMENT_ARG_ABOUT = i18n.translate(
   { defaultMessage: 'A comment to go along with the action' }
 );
 
-export const getEndpointResponseActionsConsoleCommands = (
-  endpointAgentId: string
-): CommandDefinition[] => {
+const DISABLED_COMMAND_INFO = i18n.translate(
+  'xpack.securitySolution.endpointConsoleCommands.suspendProcess.disabledCommandInfo',
+  { defaultMessage: 'This endpoint does not support this commmand' }
+);
+
+const DISABLED_COMMAND = i18n.translate(
+  'xpack.securitySolution.endpointConsoleCommands.suspendProcess.disabledCommand',
+  { defaultMessage: 'Disabled' }
+);
+
+export const DisabledTooltip = React.memo(() => {
```

Comment:
Not sure if this is the right place to have a component. I know you have converted this `ts` file into a `tsx` but what about having this in a separated file and moving the logic in the `about` field outside too? For example, where the `about` is displayed.

Reply:
FYI: @parkiino and I spoke briefly and I suggested she move the `about` text/ReactNode to a function that can just be called from here.